portal-ui/src/screens/Console/Tenants/TenantDetails/pvcs/PVCDescribe.tsx

```diff
@@ -0,0 +1,247 @@
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { Theme } from "@mui/material/styles";
+import createStyles from "@mui/styles/createStyles";
+import withStyles from "@mui/styles/withStyles";
+import {
+  actionsTray,
+  buttonsStyles,
+  hrClass,
+  searchField,
+} from "../../../Common/FormComponents/common/styleLibrary";
+import { Box } from "@mui/material";
+import Grid from "@mui/material/Grid";
+import Chip from "@mui/material/Chip";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import { setErrorSnackMessage } from "../../../../../actions";
+import { ErrorResponseHandler } from "../../../../../common/types";
+import api from "../../../../../common/api";
+import { AppState } from "../../../../../store";
+import LabelValuePair from "../../../Common/UsageBarWrapper/LabelValuePair";
+
+interface IPVCDescribeProps {
+  tenant: string;
+  namespace: string;
+  pvcName: string;
+  propLoading: boolean;
+  setErrorSnackMessage: typeof setErrorSnackMessage;
+  loadingTenant: boolean;
+}
+
+interface Annotation {
+  key: string;
+  value: string;
+}
+
+interface Label {
+  key: string;
+  value: string;
+}
+
+interface DescribeResponse {
+  annotations: Annotation[];
+  labels: Label[];
+  name: string;
+  namespace: string;
+  status: string;
+  storageClass: string;
+  capacity: string;
+  accessModes: string[];
+  finalizers: string[];
+  volume: string;
+  volumeMode: string;
+}
+
+interface IPVCDescribeSummaryProps {
+  describeInfo: DescribeResponse;
+}
+
+interface IPVCDescribeAnnotationsProps {
+  annotations: Annotation[];
+}
+
+interface IPVCDescribeLabelsProps {
+  labels: Label[];
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    ...actionsTray,
+    ...buttonsStyles,
+    ...searchField,
+    ...hrClass,
+    actionsTray: {
+      ...actionsTray.actionsTray,
+      padding: "15px 0 0",
+    },
+  });
+
+const twoColCssGridLayoutConfig = {
+  display: "grid",
+  gridTemplateColumns: { xs: "1fr", sm: "2fr 1fr" },
+  gridAutoFlow: { xs: "dense", sm: "row" },
+  gap: 2,
+  padding: "15px",
+};
+
+const HeaderSection = ({ title }: { title: string }) => {
+  return (
+    <Box
+      sx={{
+        borderBottom: "1px solid #eaeaea",
+        margin: 0,
+        marginBottom: "20px",
+      }}
+      >
+      <h3>{title}</h3>
+    </Box>
+    
+  );
+};
+
+const PVCDescribeSummary = ({describeInfo}: IPVCDescribeSummaryProps) => {
+  return (
+    <React.Fragment>
```

Comment:
Please replace this to use Fragment only

```suggestion
    <Fragment>
```